docker/ui/Dockerfile

```diff
@@ -1,38 +1,41 @@
 #
 # conductor:ui - Netflix conductor UI
 #
-FROM node
-
+FROM node:alpine
 MAINTAINER Netflix OSS <conductor@netflix.com>
 
-# Make app folders
-RUN mkdir -p /app/config /app/logs /app/libs
+# Install the required packages for the node build
+# to run on alpine
+RUN apk update && apk add \
+  autoconf \
+  automake \
+  libtool \
+  build-base \
+  libstdc++ \
+  gcc \
+  abuild \
+  binutils \
+  nasm \
+  libpng \
+  libpng-dev \
+  libjpeg-turbo \
+  libjpeg-turbo-dev 
```

Comment:
Some of these might be redundant.  We might be able to make this image smaller yet by installing only what is necessary to run the ui.